cogs/member_stats.py

```diff
@@ -0,0 +1,38 @@
+from discord.ext import commands
+from pathlib import Path
+from .validation import is_moderator, is_mod_commands_channel
+import json
+
+class MemberStats(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def mstats(self, ctx):
```

Comment:
The current command convention is `$m`, and then underscores representing spaces. So this would be `$m_stats`. Apologies for not being clear on that.